var/spack/repos/builtin/packages/lordec/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Lordec(Package):
+    """LoRDEC is a program to correct sequencing errors in long reads from
+    3rd generation sequencing with high error rate, and is especially
+    intended for PacBio reads."""
+
+    homepage = "http://www.example.com"
+    url      = "https://gite.lirmm.fr/lordec/lordec-releases/uploads/e3116a5f251e46e47f7a3b7ddb2bd7f6/lordec-src_0.8.tar.gz"
+
+    version('0.8', 'd5a57db681d1cd31c516aa780ce53f9d')
+
+    depends_on('boost')
+    depends_on('cmake@3.1.0:')
+
+    def install(self, spec, prefix):
+        make('clean', 'all', '-j8')
```

Comment:
The `-j8` shouldn't be hardcoded. Spack already adds `-j + # of cores` for you. You can override this using `spack install -j8 lordec`.